<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>887</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>40</y>
      <width>161</width>
      <height>451</height>
     </rect>
    </property>
    <property name="title">
     <string>Figures</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>151</width>
       <height>411</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="rectangleButton">
        <property name="text">
         <string>Rectangle</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="circleButton">
        <property name="text">
         <string>Circle</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="squareButton">
        <property name="text">
         <string>Square</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="hexagonButton">
        <property name="text">
         <string>Hexagon</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="ellipseButton">
        <property name="text">
         <string>Ellipse</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rhombButton">
        <property name="text">
         <string>Rhomb</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="fivePButton">
        <property name="text">
         <string>Five-pointed star</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="sixPButton">
        <property name="text">
         <string>Six-pointed star</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="eightPButton">
        <property name="text">
         <string>Eight-pointed star</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="triangleButton">
        <property name="text">
         <string>Triangle</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="parallButton">
        <property name="text">
         <string>Parallelogram</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>861</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>521</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QPushButton" name="rotateLeftButton">
        <property name="text">
         <string>Rotate left</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="rotateRightButton">
        <property name="text">
         <string>Rotate right</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="perimeterButton">
        <property name="text">
         <string>Perimeter</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="zoomOutButton">
        <property name="text">
         <string>Zoom out</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="areaButton">
        <property name="text">
         <string>Area</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="zoomInButton">
        <property name="text">
         <string>Zoom in</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="centerMassButton">
        <property name="text">
         <string>Center of mass </string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="changeSizeButton">
        <property name="text">
         <string>Change size</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>20</y>
       <width>259</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="2">
       <widget class="QPushButton" name="moveRightButton">
        <property name="text">
         <string>-&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="moveLeftButton">
        <property name="text">
         <string>&lt;-</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="moveDownButton">
        <property name="text">
         <string>↓</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="moveUpButton">
        <property name="text">
         <string>↑</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>691</width>
      <height>471</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="drawFigureButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>10</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw figure</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>887</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
